dag-server-test-values.yaml

```diff
@@ -0,0 +1,8 @@
+# !!!!!!!!!!!!!!!!!
+#
+# TEMP FILE FOR TESTING. DELETE BEFORE MERGING TO MASTER
+#
+# !!!!!!!!!!!!!!!!!
+
+airflow:
+  executor: CeleryExecutor
```

Comment:
This file should be deleted before merging.

Reply:
Im deleting this file to merge this change to master